Dockerfile

```diff
@@ -110,6 +110,8 @@ COPY quickstart.sh .
 # to work. It matches what's set in dockerfile_exec.sh.
 RUN mkdir -p local && echo '{ "static": "static_root", "debug": false, "host": "_", "https": false }' > local/environment.json
 RUN python3.6 manage.py collectstatic --noinput
+RUN python3.6 manage.py makemigrations
```

Comment:
We should not need to have make migrations in docker file. All migrations should exisyed in code and results be tested prior to any deployment.